guides/common/modules/proc_enabling-the-dhcp-module.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Enabling_the_DHCP_Module_{context}"]
+= Enabling the DHCP Module
+
+.Procedure
+. Enable the DHCP module and specify a provider in `dhcp.yml`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:enabled: https
+:use_provider: _My_DHCP_Provider_Plugin_
+----
+. If you use a remote DHCP server, change the address:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:server: 127.0.0.1
+----
++
+Note that if the DHCP server is running remotely, some providers (notably ISC) require that the configuration files must be accessible to the {SmartProxyServer}.
+You can use network file systems to achive this.
+. Optional: By default, you cna manage all available subnets.
+To mitigate the performance penalty from loading all available subnets, restrict DHCP to a list of subnets to optimize performace in `_My_Network_Address_/_My_Network_Mask_` notation:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:subnets: [192.168.205.0/255.255.255.128, 192.168.205.128/255.255.255.128]
+----
+. Optional: Adjust the configuration in the DHCP-provider specific configuration file.
```

Comment:
Interestingly enough `guides/common/modules/proc_configuring-satellite-deployment-with-an-external-dhcp-server.adoc` doesn't document `--foreman-proxy-dhcp-subnets`. The equivalent of above would be:
```
--foreman-proxy-dhcp-subnets 192.168.205.0/255.255.255.128 \
--foreman-proxy-dhcp-subnets 192.168.205.128/255.255.255.128
```

I'm also very big on using RFC5735 IP ranges in documentation. See https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600

Reply:
Added to "DHCP Options for Network Configuration" reference.